.github/actions/verify-token/src/utils.ts

```diff
@@ -1,4 +1,9 @@
 import { githubObj } from "./types";
+import * as core from "@actions/core";
+import * as sigstore from "sigstore";
```

Comment:
nit: avoid the `sigstore.sigstore` stuff.

```suggestion
import { sigstore } from "sigstore";
```